### PR TITLE
Added Configurations to config file

### PR DESCRIPTION
**Pull request by betterclever**
Fixes Issue #32: Added configurable parameters to config file.

### CODE REVIEW

**Comment by betterclever** [comment]
@chiragw15 @mayank408 @rishiraj824 @chashmeetsingh Please review.

**Comment by betterclever** [comment]
@saurabhjn76 @uday96 Please review.

File: app/main.py [hunk 1]
```diff
@@ -114,10 +126,13 @@ def close_stream():
...
 # TODO: Decide threshold by a training based system.
 # adjust threshold manually for now.
-sphinxRecognizer = SphinxRecognizer(threshold=1e-23)
+sphinxRecognizer = SphinxRecognizer(threshold=1e-30)
 
+val = 1
 while True:
     buffer = stream.read(20480, exception_on_overflow=False)
+    val += 1
+    print(val)
```
Comment (chashmeetsingh) [inline app/main.py]: Do we need print statements here? Shouldn't they be removed?

File: app/speech/SphinxRecognizer.py [hunk 2]
```diff
@@ -9,28 +8,29 @@ def __init__(self, threshold, sample_rate=16000):
...
-        #self.decoder.set_search('keyphrase')
+        self.decoder = pocketsphinx.Decoder(self.create_config(dict_name))
 
     def create_config(self, dict_name):
-        modeldir = "/usr/lib/python3.6/site-packages/pocketsphinx/model"
-        config = Decoder.default_config()
-        config.set_string('-hmm', os.path.join(modeldir, 'en-us'))
+        pocketsphinx_dir = os.path.dirname(pocketsphinx.__file__)
+        model_dir = os.path.join(pocketsphinx_dir, "model")
+        print(model_dir)
```
Comment (chashmeetsingh) [inline app/speech/SphinxRecognizer.py]: Here too.

Comment (betterclever) [inline app/speech/SphinxRecognizer.py]: Yes, they are not needed. I used them for debugging. Removing them. Thanks for pointing out.

**Comment by betterclever** [comment]
@chashmeetsingh Done.

**Pull request merged**